src/contracts/uniswap/UniswapV2Library.sol

```diff
@@ -4,6 +4,7 @@
 // - Added appropriate SPDX license comment
 // - Modified Solidity version
 // - Formatted code
+// - Shortened revert messages
```

Comment:
So that they fit in one word?

Reply:
Yes. We have a lint rule for this. I think Solidity may generate an `mstore` instead of a `codecopy` for short strings.